tests/BUILD

```
@@ -54,6 +54,7 @@ py_test(
     deps = [
         "//jax",
         "//jax:experimental_array_api",
+        "//jax:internal_test_util",
```

Sorry, I was mistaken: we need `jax:test_util` instead:
```suggestion
        "//jax:test_util",
```